components/infobox/commons/infobox_map.lua

```diff
@@ -0,0 +1,52 @@
+local BasicInfobox = require('Module:Infobox/Basic')
+local Class = require('Module:Class')
+local Cell = require('Module:Infobox/Cell')
+local Namespace = require('Module:Namespace')
+local getArgs = require('Module:Arguments').getArgs
+
+local Map = Class.new(BasicInfobox)
+
+function Map.run(frame)
+	local map = Map(frame)
+	local args = getArgs(frame)
+	local infobox = map.infobox
+	infobox:name(Map:getNameDisplay(args))
+	infobox:image(args.image, args.defaultImage)
+	infobox:centeredCell(args.caption)
+	infobox:header('Map Information', true)
+	infobox:fcell(Cell:new('Creator'):options({makeLink = true}):content(
+		args.creator or args['created-by']):make())
+	Map:addCustomCells(infobox, args)
+	infobox:bottom(Map.createBottomContent(infobox))
+
+	if Namespace.isMain() then
+		infobox:categories('Maps')
+		Map:_setLpdbData(args)
+	end
+
+	return infobox:build()
+end
+
+--- Allows for overriding this functionality
+function Map:getNameDisplay(args)
+	return args.name
+end
+
+--- Allows for overriding this functionality
+function Map:addToLpdb(lpdbData, args)
+	return lpdbData
+end
+
+function Map:_setLpdbData(args)
+	local lpdbData = {
+		name = self.name,
+		type = 'map',
+		image = args.image,
+	}
+
+	lpdbData = self:addToLpdb(lpdbData, args)
+	lpdbData.extradata = mw.ext.LiquipediaDB.lpdb_create_json(lpdbData.extradata or {})
+	mw.ext.LiquipediaDB.lpdb_datapoint('map_TEST_' .. lpdbData.name, lpdbData)
```

Comment:
What's with the TEST?

Please make consistent with https://liquipedia.net/commons/Template:MapData

Reply:
ah damn forgot to remove the `TEST_` (was so i could compare the lpdb entries)